config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   root 'users#index'
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts do 
+      resources :comments, only: [:create]
```

Comment:
**You did an awesome job!**
We have two more requirements to complete. Let's get started. 

- [ ] For starting, your comments should reponse to these routes: In your project I can only see posts#create, please posts#new route as well. Thank you!
```ruby
GET   /posts/new                   posts#new
  POST   /posts(.:format)                     posts#create
```

Reply:
Hello @Rustamxon7. Thanks for the prompt feedback. I'm unclear about this requirement. What I understood from the [docs](https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions) is that ```resources :posts do```  creates all routes, including post#create and post#new. Are you referring to the comments route (``` resources :comments, only: [:create]```) instead?

Reply:
No, your posts have all routes that we need, what we need not in your comments add :new in only: [...]. You can check by `rails routes` to see all routes you have.

Reply:
Understood. Thank you.